apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/project.json

```diff
@@ -25,6 +24,7 @@
           "type": "platform"
         }
       }
-    }
+    },
+    "net451": {}
```

Comment:
Yes, there appears to be some bug with VS where it won't load the tests into the test explorer if they are in the other order.